guides/development-workflow.md

```diff
@@ -456,31 +525,44 @@ When we have finished a piece of work on a branch, we
 [make a pull request](https://help.github.com/articles/using-pull-requests/)
 using the project's GitHub/GitLab page.
 
-We prefix pull request titles with the Story ID to make it easy to find the
-story and the acceptance criteria that should be met.
+We link each pull request with the related Trello card to make it easier to
+find. We could prefix the card number to the pull request title or link to the
+card directly within the description or a comment.
 
 A good pull request should:
 
-- be created by the authoring developer
-- meet all of the acceptance criteria on the associated story
-- focus on the single problem at hand, not doing so will usually increase the
-  time taken to get it merged
-- include appropriate detail to assist the reviewer, this includes links to
-  relevant content and a duplication of anything of note in the commit
-  descriptions
-- update the [changelog](#tracking-changes) if it's a significant change
-
-```
-# Poor
-A new registration form and fix a bug with contacts
-
-# Great
-[123456789] New users can now register for an account
-
-- This has been rate limited, would like your feedback on the values I've chosen
-- Passwords are encrypted using BCrypt, so I've taken this opportunity to
-  increase the default rounds from 8 to 12
-```
+- link to the Trello card for extra context, do not rely on Trello cards alone
+  as they can be easily removed or moved
+- describe the problem without assuming the reader has existing context
+- highlight the changes being made
+- describe and bring attention to particular difficulties or areas that you
+  particularly want the opinion of the reviewer
+- include screenshots for user facing changes (following the pull request
+  template prompts)
+- be clear which acceptance criteria are being met (or if it's a prior
+  refactoring)
+- explain if a task is being broken down into multiple pull requests and what
+  the review order should be
+- be focused on a single problem, preferring to split additional problems into
+  other pull requests, the smaller the pull request, the easier it is to be
+  reviewed and the quicker it is likely to be merged
+- updates the [changelog](#tracking-changes) if it's a significant change
```

Comment:
Worth making sure we're consistent between review and author for this. Is it only "significant changes" or any change (as implied by the review guide)?